go.mod

```diff
@@ -5,6 +5,7 @@ go 1.22
 require (
 	connectrpc.com/connect v1.16.2
 	github.com/Masterminds/semver/v3 v3.2.1
+	github.com/Masterminds/squirrel v1.5.4
```

Comment:
Would like to discuss, if we really want to add this as a dependency for such simple queries

Reply:
in go-ipam we simply use `github.com/jmoiron/sqlx` without any additional abstraction

Reply:
It's removed now.